src/cms/models/offers/offer_template.py

```diff
@@ -32,7 +31,7 @@ class OfferTemplate(models.Model):
         verbose_name=_("URL"),
         help_text=_("This will be an external API endpoint in most cases."),
     )
-    post_data = JSONField(
+    post_data = models.JSONField(
```

Comment:
This is awesome. I think that was the main blocker for using SQLite in Dev model.

Reply:
For `Region.push_notification_channels` (Array of strings), I think we should just go for a normal ForeignKey to the `PushNotificationChannel` model instead of `ArrayField`...
For `RecurrenceRule.weekdays_for_weekly` (Array of ints) we can maybe also use a `JSONField`?

Reply:
Let's continue this discussion in #457